openvdb_points_houdini/houdini/SOP_OpenVDB_Points.cc

```diff
@@ -593,11 +593,23 @@ newSopOperator(OP_OperatorTable* table)
         .setDefault("points")
         .setHelpText("Output grid name."));
 
+    parms.add(hutil::ParmFactory(PRM_TOGGLE, "automatic", "Automatic Voxel Size")
+        .setDefault(PRMzeroDefaults)
+        .setHelpText("*BETA* When enabled, automatically attempts to calculate a "
```

Comment:
Yep, remove "*BETA*". I'd also suggest phrasing it as calculating the voxel size. If you've provided a transform, it's really just computing the best voxel size for this transform. Suggest we enable it by default as well.